tests/test_motifs.py

```diff
@@ -112,8 +112,8 @@ def test_motifs_max_matches():
     # This test covers the following:
 
     # A time series contains motif A at four locations and motif B at two.
-    # If `max_motifs=2` the result should contain only the top two matches of motif A
-    # and the top two matches of motif B as two separate motifs.
+    # If `max_moitf=2` and `max_matches=2`, the result should contain two
```

Comment:
Fixing a few spelling mistakes and changing the numbers, I think we should change the language to say:

"If `max_motifs=2` and `max_matches=3`, the result should contain (at most) two sets of motifs and each motif set should contain (at most) the top three matches"

I like to make the numbers different so as not to imply that there is any significance in the numbers both being the same (i.e., 2)

Reply:
Of course, you'll need to change `max_matches` to `3` in the actual code

Reply:
@seanlaw 
I see your point. I will revise it. 

Just as a heads up, it is possible that we get three matches instead of two as the number of returned matches depends on the `max_distance` as well. If I notice such thing, I will motidy the input time series `T`.